neuromation/api/admin.py

```diff
@@ -131,8 +132,9 @@ async def set_user_quota(
         self,
         cluster_name: str,
         user_name: str,
-        gpu_value_minutes: Optional[float],
-        non_gpu_value_minutes: Optional[float],
```

Comment:
Quotas are ints everywhere (on server, in cli parser), so I think `Optional[float]` was a mistake.